src/main/java/me/totalfreedom/totalfreedommod/command/Command_gtfo.java

```diff
@@ -54,6 +54,9 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         // Rollback
         plugin.rb.rollback(player.getName());
 
+        // Rollback with CoreProtect
```

Comment:
yes- you need BlocksHub along with CoreProtect though, but yes it works. BlocksHub is so that CoreProtect logs AsyncWorldEdit actions.